cleanlab_studio/cli/dataset/upload_helpers.py

```diff
@@ -153,7 +153,7 @@ def validate_and_process_record(
                         ValidationWarning.TYPE_MISMATCH,
                     )
             elif col_feature_type == FeatureType.filepath:
-                if schema.metadata.modality == Modality.image:
+                if schema.metadata.modality == Modality.image and not disable_file_existence_check:
```

Comment:
while we **do** implicitly know that the file exists for zip uploads, we don't know that the image file is readable. that check would still need to occur

Reply:
Do you think given our discussions about moving towards lazier validation, we still need to perform this check?

Reply:
OK, fair point. Let's skip.

I would rename the flag though (we're not just disabling the file existence check, we're also disabling the image readability check)